src/js/components/covid19/DataSourcesAndMethodologiesPage.jsx

```diff
@@ -197,6 +193,7 @@ export default () => {
             <main id="main-content" className="main-content">
                 <div className={`sidebar usda__flex-col${dataDisclaimerBanner !== 'hide' ? ' covid-sidebar-banner' : ''}`}>
                     <Sidebar
+                        isGoingToBeSticky
                         pageName="data-sources"
                         fixedStickyBreakpoint={getStickyBreakPointForSidebar(Cookies.get(cookie))}
```

Comment:
This fn doesnt actually use this input. Should it?

Reply:
it should probably take an array of cookies but is not currently necessary.